[2024-04-19T14:33:36.110+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_data_pipeline.run_power_data_manipulation manual__2024-04-19T14:33:28.923740+00:00 [queued]>
[2024-04-19T14:33:36.121+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_data_pipeline.run_power_data_manipulation manual__2024-04-19T14:33:28.923740+00:00 [queued]>
[2024-04-19T14:33:36.122+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 3
[2024-04-19T14:33:36.141+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): run_power_data_manipulation> on 2024-04-19 14:33:28.923740+00:00
[2024-04-19T14:33:36.148+0000] {standard_task_runner.py:60} INFO - Started process 75 to run task
[2024-04-19T14:33:36.152+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'my_data_pipeline', 'run_power_data_manipulation', 'manual__2024-04-19T14:33:28.923740+00:00', '--job-id', '325', '--raw', '--subdir', 'DAGS_FOLDER/my_data_pipeline.py', '--cfg-path', '/tmp/tmplx9m70df']
[2024-04-19T14:33:36.154+0000] {standard_task_runner.py:88} INFO - Job 325: Subtask run_power_data_manipulation
[2024-04-19T14:33:36.245+0000] {task_command.py:423} INFO - Running <TaskInstance: my_data_pipeline.run_power_data_manipulation manual__2024-04-19T14:33:28.923740+00:00 [running]> on host a1c7b9609883
[2024-04-19T14:33:36.375+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_data_pipeline' AIRFLOW_CTX_TASK_ID='run_power_data_manipulation' AIRFLOW_CTX_EXECUTION_DATE='2024-04-19T14:33:28.923740+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-19T14:33:28.923740+00:00'
[2024-04-19T14:33:36.378+0000] {logging_mixin.py:188} INFO - No TXT file found in the directory
[2024-04-19T14:33:36.379+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-04-19T14:33:36.391+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=my_data_pipeline, task_id=run_power_data_manipulation, execution_date=20240419T143328, start_date=20240419T143336, end_date=20240419T143336
[2024-04-19T14:33:36.446+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-19T14:33:36.470+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
